docs/spec/v1beta3/providers.md

```diff
@@ -945,6 +948,32 @@ metadata:
 stringData:
     address: https://username:password@<alertmanager-url>/api/v2/alerts/"
 ```
+Bearer Token Authentication:
+Create a Secret with [the `token`](#token-example), and a `alertmanager` Provider with a [Secret
+reference](#secret-reference) and Prometheus Alertmanager [HTTP API
+URL](https://prometheus.io/docs/alerting/latest/https/#http-traffic) set directly in the spec.address field.
+
+```yaml
+---
+apiVersion: notification.toolkit.fluxcd.io/v1beta3
+kind: Provider
+metadata:
+  name: alertmanager
+  namespace: default
+spec:
+  type: alertmanager
+  address: <alertmanager-url>/api/v2/alerts/
```

Comment:
```suggestion
  address: https://<alertmanager-hostport>/api/v2/alerts/
```

Would you please kindly also fix this in the basic auth example? :pray: Much appreciated!

`https://username:password@<alertmanager-url>/api/v2/alerts/"`

Becomes:

`https://<username>:<password>@<alertmanager-hostport>/api/v2/alerts/`